src/app/utils/__tests__/mapFormDataToApiData.test.ts

```diff
@@ -98,7 +103,7 @@ describe('mapFormDataToApiData', () => {
     });
 
     it("should set 'medlemskap.har_bodd_i_utlandet_siste_12_mnd' in api data correctly", () => {
-        expect(resultingApiData.medlemskap.harBoddIUtlandetSiste12Mnd).toBe(true);
+        expect(resultingApiData.medlemskap.harBoddIUtlandetSiste12Mnd).toBe(false);
```

Comment:
Denne testen kan sikkert slettes fra denne fila, og heller testes i selve testen for getMedlemskapApiDataFromSøknadsdata